src/IECoreHoudini/plugin/Plugin.cpp

```diff
@@ -193,19 +193,27 @@ void newRenderHook( GR_RenderTable *table )
 
 void newGeometryPrim( GA_PrimitiveFactory *factory )
 {
-
 	GA_PrimitiveDefinition *primDef = factory->registerDefinition(
-		CortexPrimitive::typeName, CortexPrimitive::create,
-		GA_FAMILY_NONE, ( std::string( CortexPrimitive::typeName ) + "s" ).c_str()
+		CortexPrimitive::typeName,
+#if UT_MAJOR_VERSION_INT >= 16
+		CortexPrimitive::createCortexPrimBlock,
```

Comment:
This name seems to be our choice. Lets keep calling it `create()` since the older `create()` is excluded from the H16 compilation.